test/unix/test_lwt_engine.ml

```diff
@@ -6,28 +6,6 @@
 open Test
 open Lwt.Infix
 
-let selection_tests = [
-  test "libev: default when enabled in build bot"
-    (fun () ->
-      if not Lwt_config._HAVE_LIBEV then Lwt.return_true
-      else
-        (* Check if this is running inside Travis or AppVeyor. *)
-        let in_travis =
-          try ignore (Sys.getenv "TRAVIS_COMMIT"); true
-          with Not_found -> false
-        in
-
-        let in_appveyor =
-          try ignore (Sys.getenv "APPVEYOR_REPO_COMMIT"); true
-          with Not_found -> false
-        in
-
-        if not (in_travis || in_appveyor) then Lwt.return_true
-        else Lwt.return Lwt_config.libev_default);
```

Comment:
Instead of deleting this, you could check whether the `CI` env var is `true` or `True`, or check whether `GITHUB_ACTIONS` is `true`.

Reply:
I wasn't quite sure of the meaning of this test. Since `in_travis` and `in_appveyor` would always be false, it's basically a noop. I can do what you propose, and then the meaning of the test would be "if we're in github actions, then check that `Lwt_config.libev_default` is true". Is that what we want the test to check (if so, why do we want to test that?)?

Reply:
It's probably left-over from special-casing for different CIs and I think we can remove it.